### DISCUSSION TITLE
the library needs a better name

### DISCUSSION THREAD

**Discussion by danwinship** [cilium/ebpf#798] [Ideas]
It is hard to refer to this library in a discussion... You obviously can't just call it "ebpf". The string "`cilium/ebpf`" is theoretically unambiguous, but in actual usage it's _quite_ ambiguous (and tends to imply "the eBPF code that is part of Cilium" rather than "the eBPF library that is produced by the Cilium project separately from Cilium". You end up having to say something like "Cilium's golang eBPF loader implementation" which is way too long...

So anyway, a more codename-y name would be cool...

**Comment by ti-mo** [comment]
I've been referring to the project as 'ebpf-go' for a while, would that fit the bill?

**Reply by danwinship** [reply]
Yeah, that would work. It's maybe a bit close to "[libbpfgo](https://github.com/aquasecurity/libbpfgo)", but distinct enough to make it clear that it's not the same thing.
